### ISSUE TITLE
Create how-to describing implementation of DA layer client

### ISSUE THREAD

**Issue by tzdybal** [rollkit/rollkit#74]
It is important to describe how to implement new Data Availability layer client and use it with Optimint.
It can be a tutorial, step-by-step guide, detailed example or even very precise package-level godoc.

**Comment by Manav-Aggarwal** [comment]
Blocked on: https://github.com/rollkit/rollkit/issues/1122

**Comment by nashqueue** [comment]
Duplicate of https://github.com/rollkit/docs/issues/253
